packages/kit/src/runtime/server/page/render.js

```diff
@@ -216,7 +216,7 @@ function serialize_error(error) {
 	let serialized = try_serialize(error);
 	if (!serialized) {
 		const { name, message, stack } = error;
-		serialized = try_serialize({ name, message, stack });
+		serialized = try_serialize({ ...error, name, message, stack });
```

Comment:
this is a weird way to write this. At this point you might as well just pass `error` instead of `{ ...error, name, message, stack }`

Reply:
Passing error does not work because try_serialize doesn't detect it's type in development (which is your custom class prototype).  `{ ...error }` would be possible though. That how I understand it at least.

Reply:
```js
class MyError extends Error {
	test = "test"
}

var error = new MyError();
var { name, message, stack } = error;
console.log({ ...error, name, message, stack })
```
removing `name, message, stack` doesn't work because JavaScript is strange. Also `JSON.stringify(error)` doesn't work for the same reason AFAICT.

Reply:
Ah, it looks like it's because `Error`'s default properties are not enumerable. Though if the Svelte compiler throws an error it you can see the following properties: `stack,message,name,code,start,end,pos,filename,frame,plugin,id,pluginCode,loc`.

Anyway, your change looks good to me